packages/core/helper-plugin/src/components/FilterListURLQuery.tsx

```diff
@@ -29,7 +51,7 @@ export const FilterListURLQuery = ({ filtersSchema }) => {
   };
 
   return (
-    query?.filters?.$and.map((filter, i) => {
+    query?.filters?.$and?.map((filter: Filter, i: number) => {
```

Comment:
```suggestion
    query?.filters?.$and?.map((filter, i) => {
```

You should not need to type array methods because the values are inferred, can you double check you're not typing things you shouldn't be please? :)